frontend/public/locales/en/translation.json

```diff
@@ -1517,6 +1520,7 @@
     "managedClusterSet.form.removed": "Removed",
     "managedClusterSet.form.transferred": "Transferred",
     "managedClusterSet.form.unchanged": "No change",
+    "Managing hosts with infrastructure environments makes it easy to set settings across multiple hosts.": "Managing hosts with infrastructure environments makes it easy to set settings across multiple hosts.",
```

Comment:
opinion: I understand the benefits of having "real texts" as keys for translations but I would shorten them for similarly long cases

Reply:
I disagree.
How do we say which key is too long ?
Also it does not really matter how this file looks like, its not something we would be looking at. The important part, which needs to be readable, is in the actual code in which case a using a real text is a clear winner over some key